scipy/stats/tests/test_stats.py

```diff
@@ -5644,19 +5690,35 @@ def test_alternatives(self, alt, pr, tr):
         assert_allclose(statistic, tr, atol=1e-10)
 
     def test_errors_unsupported(self):
-        # confirm that attempting to trim with NaNs or permutations raises an
-        # error
-        match = "Permutations are currently not supported with trimming."
-        with assert_raises(ValueError, match=match):
+        # confirm that attempting to trim with permutations raises an error
+        match = "Use of `permutations` is incompatible with with use of `trim`."
+        with assert_raises(NotImplementedError, match=match):
             stats.ttest_ind([1, 2], [2, 3], trim=.2, permutations=2)
 
+    @array_api_compatible
+    @pytest.mark.skip_xp_backends(cpu_only=True,
```

Comment:
The top of this file has `skip_xp_backends = pytest.mark.skip_xp_backends`, if you'd like to reduce verbosity a tad

Reply:
Ok. 
I think I saw a PR in which you tried to eliminate the need for `usefixture` line everywhere. Did that not work out?

Reply:
hmm, there is gh-20368, but that isn't quite what it was trying to do - `usefixtures` is still needed wherever you are trying to skip. In any case, I couldn't get that PR working.

Reply:
at the minute, you have to wait until every test in a test file is converted, then the verbosity can be reduced greatly with `pytestmark = [array_api_compatible, pytest.mark.usefixtures("skip_xp_backends")]`

Reply:
In this sense, there would be benefits to having smaller test files than `test_stats.py`. I suppose keeping the commit history is more important, though.

Reply:
Hah yeah, I'd love to have smaller files. I did find a way of splitting the files while preserving the history. But that's probably better done when things are a bit more stable.